src/cli/cli.cc

```diff
@@ -3648,6 +3731,86 @@ int main (const int argc, const char* argv[]) {
     }
   });
 
+  createSubcommand("setup", { "--platform", "--yes", "-y" }, false, [&](const std::span<const char *>& options) -> void {
+    auto win = platform.win;
+
+    auto help = false;
+    String targetPlatform;
+    auto targetAndroid = false;
+    auto targetWindows = false;
+    auto yes = false;
+    String yesArg;
+    for (auto const arg : options) {
+      if (is(arg, "-h") || is(arg, "--help")) {
+        help = true;
+      }
+
+      auto platform = optionValue("setup", arg, "--platform");
+      if (platform.size() > 0) {
+        targetPlatform = platform;
+      }
+
+      if (is(arg, "-y") || is(arg, "--yes")) {
+        yes = true;
+      }
+
+      if (help) {
+        break;
```

Comment:
in this case it should just exit with 0 IMHO
```cpp
     printHelp("setup");
     exit(0);
```

Reply:
maybe just after
```cpp
if (is(arg, "-h") || is(arg, "--help")) {
```

Reply:
@jwerle Can you tie break this one please?

Reply:
@chicoxyzzy Sorry I get it now. Fixed I think.